mapper/blockers.lua

```diff
@@ -56,3 +42,8 @@ function trigger_func_mapper_blockers_blockers_block()
     amap:terminate_walker()
 end
 
+function trigger_func_blockers_unblockable()
+    amap.blockers:set_blockable(false)
+end
+
+amap.blockers:init()
```

Comment:
Czy to nie powinno byc uwarunkowane czy juz jest zainicjowane?

Reply:
Init może dziać się zawsze. Jeżeli byłby warunek to dałbym go w `init`